command_line/sim_view.py

```diff
@@ -808,14 +858,14 @@ def _reset(self, _press=None):
         "DomainSize":[6, 200, 2, 10, 30],
         "MosAngDeg":[0.01, 5, 0.01, 0.1, 0.1001],
         "ucell_scale":[0.5, 2., 0.05, 0.1, 1],
-        "Diff_gamma":[1, 1000, 1, 10, 50],
-        "Diff_sigma":[.001, 5, .1, 1, .1601],
-        "Diff_aniso":[.01, 10, .01, .1, 2],
+        "Diff_gamma":[1, 300, 1, 10, 50],
+        "Diff_sigma":[.01, 0.7, .01, 0.05, .4],
+        "Diff_aniso":[.01, 10, .1, 1, 3],
         "Energy":[6500, 12000, 10, 30, 9500],
         "Bandwidth":[0.01, 5.01, 0.1, 1, 0.31],
-        "RotX": [-180, 180, 0.01, 0.1, 0],
-        "RotY": [-180, 180, 0.01, 0.1, 0],
-        "RotZ": [-180, 180, 0.01, 0.1, 0],
+        "RotX": [-180, 180, 0.1, 1., 0],
```

Comment:
For SFX purposes I think we do care about misorientations of 0.01 degrees and would want to keep the small step size this small. Open to negotiation on the big step size. @nksauter please contradict me if this doesn't match your understanding.